libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -86,7 +89,9 @@ function parseCommandOutput(payload: InvokeCommandOutput): any | undefined {
       throw newError;
     }
     throw new Error(
-      `Invoke failed with message: "${payload.FunctionError}". Full error: "${errorText}"`
+      `Invoke failed with message: "${
+        payload.FunctionError
+      }". Full error: "${errorText}" (logs: ${cloudwatchLogsPath(functionArn)})`
```

Comment:
imo the logs should go before the full error, they'll probably be much more useful

```suggestion
      `Invoke failed with message: "${
        payload.FunctionError
      }".\nLogs: ${cloudwatchLogsPath(functionArn)}\nFull Error: "${errorText}"`
```